admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +168,139 @@ A failure from this check indicates that a significant amount of space in etcd
+exceeds 40 GB.
+
+|`etcd_traffic`
+|This check detects higher than normal traffic on an etcd host. It fails if a
+`journalctl` log entry with an etcd sync duration warning is found.
+
+For further information on improving etcd performance, see
+xref:../scaling_performance/host_practices.adoc#scaling-performance-capacity-host-practices-etcd[Recommended Practices for {product-title} etcd Hosts] and the
+link:https://access.redhat.com/solutions/2916381[Red Hat Knowledgebase].
 
 |`etcd_volume`
 |This check ensures that the volume usage for an etcd cluster is below a maximum
 user-specified threshold. If no maximum threshold value is specified, it is
 defaulted to `90%` of the total volume size.
 
-A user-defined limit may be set by passing the variable
-`etcd_device_usage_threshold_percent=90` to the `openshift_health_checker` role.
+A user-defined limit may be set by passing the
+`etcd_device_usage_threshold_percent` variable.
 
 |`docker_storage`
-|Only runs on hosts that depend on the *docker* damon (nodes and containerized
+|Only runs on hosts that depend on the *docker* daemon (nodes and containerized
 installations). Checks that *docker*'s total usage does not exceed a
 user-defined limit. If no user-defined limit is set, *docker*'s maximum usage
-threshold defaults to 90% of the total size available. The threshold
-limit for total percent usage can be set with a variable in your inventory file:
-`max_thinpool_data_usage_percent=90`.
-|===
+threshold defaults to 90% of the total size available.
 
-To disable specific checks, include the variable `openshift_disable_check` with
-a comma-delimited list of check names in your inventory file. For example:
+The threshold limit for total percent usage can be set with a variable in your
+inventory file, for example `max_thinpool_data_usage_percent=90`.
 
-----
-openshift_disable_check=ovs_version,etcd_volume
-----
+|`kibana`, `curator`, `elasticsearch`, `fluentd`
+|This set of checks verifies that Elasticsearch, Fluentd, and Curator pods have
+been deployed and are in a `running` state, and that a connection can be
+established between the control host and the exposed Kibana URL. These checks
+will only run if the `openshift_hosted_logging_deploy` inventory variable is set
+to `true`, to ensure that they are executed in a deployment where a logging
+stack has been deployed.
 
+|`logging_index_time`
+|This check detects higher than normal time delays between log creation and log
+aggregation by Elasticsearch in a logging stack deployment. It fails if a
+user-defined timeout is reached before logs are able to be queried through
+Elasticsearch.
+
+A user-defined timeout may be set by passing the
+`openshift_check_logging_index_timeout_seconds` variable. For example, setting
+`openshift_check_logging_index_timeout_seconds=30` will cause the check to fail
+if a newly-created log entry is not able to be queried via Elasticsearch after
+30 seconds.
```

Comment:
Don't think we need the long variable name twice. And to Rodolfo's point:

`Users that either require lower-latency log aggregation or are comfortable with higher latency may adjust this timeout with an Ansible variable. For example, setting `openshift_check_logging_index_timeout_seconds=45` relaxes the timeout to 45 seconds.`